jobs/rootfs-builder.jpl

```diff
@@ -81,6 +96,11 @@ node("debos && docker") {
     def arch = "${params.ROOTFS_ARCH}"
     def pipeline_version =  "${params.PIPELINE_VERSION}"
     def docker_image = "${params.DOCKER_BASE}debos"
+    def rootfs_type = "${params.ROOTFS_TYPE}"
+
+    if (rootfs_type == "buildroot") {
+        docker_image = "kernelci/staging-buildroot"
```

Comment:
It'd be good to not hardcode docker image name, but rather use `${params.DOCKER_BASE}` so it can be used for staging and prod instances

```
docker_image = "${params.DOCKER_BASE}buildroot"
```